static/docker-compose.yml

```diff
@@ -28,7 +28,8 @@ services:
       - 'import requests'
       - 'print(requests.get("http://127.0.0.1:4350/api/v2/health/live").json())'
       - 'EOF'
-
+    entrypoint: npm run
+    command: "start"
```

Comment:
All other services which use the plextracapi image have their entrypoints defined in this way.  This just makes the entrypoints straightforward for devs trying to make sense of how this container is started.

Reply:
I think this is fine but I'll need to test it doesn't conflict with the base container entrypoint for any reason.